modules/ossm-deploy-multi-mesh.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/ossm-deploy-mod-v2x.adoc
+
+
+[id="ossm-deploy-multi-mesh_{context}"]
+= Multimesh or Federated deployment model
+
+[role="_abstract"]
+*Federation* is a deployment model that lets you share services and workloads between separate meshes managed in distinct administrative domains.
+
+The Istio multi-cluster model requires a high level of trust between meshes, and requires remote access to all Kubernetes API servers on which the individual meshes reside. {ProductName} federation takes an opinionated approach to a multi-cluster implementation of Service Mesh that assumes _minimal_ trust between meshes.
```

Comment:
Somehow "requires... and requires' catches the eye. Maybe one could be eliminated or the sentence can be split into 2?
And an Oxford comma is used in lists of three and more, it doesn't look justified in phrases like "model requires X, and Y". Maybe here a comma was meant for easier readability?